app/controllers/survey_controller.rb

```diff
@@ -1,4 +1,10 @@
 class SurveyController < ApplicationController
   def index
+    @questionList = Question.order(:order)
+    #@questions = [{:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}, {:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}]
+    @questionList.all.each do |theQuestion|
+      theQuestion.answers = Answer.where(question_id: theQuestion.id)
+    end
+    #puts @questionList[0].answers
```

Comment:
That was just something that printed the answers to the terminal that I was using for debugging.